draft-oid4vc-haip-sd-jwt-vc.md

```diff
@@ -205,7 +205,7 @@ This is an example of a Wallet Instance Attestation:
    * MUST support protocol extensions for SD-JWT VC credential format profile as defined in this specification (#vc_sd_jwt_profile).
    * As a way to invoke the Wallet, at least a custom URL scheme `haip://` MUST be supported. Implementations MAY support other ways to invoke the wallets as agreed by trust frameworks/ecosystems/jurisdictions, not limited to using other custom URL schemes.
    * Response type MUST be `vp_token`.
-   * Response mode MUST be `direct_post` with `redirect_uri` as defined in Section 6.2 of [@!OIDF.OID4VP].
+   * Response mode MUST be `direct_post` with `response_uri` as defined in Section 6.2 of [@!OIDF.OID4VP].
```

Comment:
```suggestion
   * Response mode MUST be `direct_post` where the verifier returns `redirect_uri` in response to the HTTP POST request from the wallet, where the wallet redirects the user to, as defined in Section 6.2 of [@!OIDF.OID4VP].
```

Reply:
You were faster than me (https://github.com/vcstuff/oid4vc-haip-sd-jwt-vc/issues/71#issuecomment-1823176043) 👍 . Definitely feels like something that should be part of this profile.

Should we mention section 11.5 as this describes the whole flow (and is really helpful if you want to implement this)?

Reply:
Ah okay thanks for the clarification, I will fix it.

Reply:
I added your suggestion and also mentioned section 11.5 for the implementation details. Please have another look and let me know if it's okay.